yarn-project/acir-simulator/src/client/private_execution.ts

```diff
@@ -154,15 +154,15 @@ export class PrivateFunctionExecution {
       },
       emitUnencryptedLog: message => {
         // https://github.com/AztecProtocol/aztec-packages/issues/885
-        const log = Buffer.concat(message.map(charBuffer => convertACVMFieldToBuffer(charBuffer).subarray(-1)));
+        const log = Buffer.concat(message.map((charBuffer: any) => convertACVMFieldToBuffer(charBuffer).subarray(-1)));
         unencryptedLogs.logs.push(log);
         this.log(`Emitted unencrypted log: "${log.toString('ascii')}"`);
         return Promise.resolve(ZERO_ACVM_FIELD);
       },
       emitEncryptedLog: ([acvmContractAddress], [acvmStorageSlot], [ownerX], [ownerY], acvmPreimage) => {
         const contractAddress = AztecAddress.fromBuffer(convertACVMFieldToBuffer(acvmContractAddress));
         const storageSlot = fromACVMField(acvmStorageSlot);
-        const preimage = acvmPreimage.map(f => fromACVMField(f));
```

Comment:
Hmm, we shouldn't need to add `any` here unless another error is causing TS to get confused

Reply:
Cleaned that up in [434954f](https://github.com/AztecProtocol/aztec-packages/pull/1016/commits/434954f952d130852bf7b61fb5679facf660bb06)

For some reason TS was complaining at some point when debugging. Seems to be ok now.